examples/refrigerator-app/silabs/data_model/refrigerator-thread-app.zap

```diff
@@ -41,16 +41,14 @@
         "code": 22,
         "profileId": 259,
         "label": "MA-rootdevice",
-        "name": "MA-rootdevice",
-        "deviceTypeOrder": 0
+        "name": "MA-rootdevice"
       },
       "deviceTypes": [
         {
           "code": 22,
           "profileId": 259,
           "label": "MA-rootdevice",
-          "name": "MA-rootdevice",
-          "deviceTypeOrder": 0
```

Comment:
I had this appear randomly on one of my zap regens, any idea where the `deviceTypeOrder ` comes from? I see we are removing it here, but some zap parameters seem to believe it needs to be added.

Reply:
i don't know to be honest. We could ask the zap team. maybe it as to do with the zap version I have. it is not the latest